test/features/interlinks/interlinks.test.js

```diff
@@ -0,0 +1,34 @@
+import { readFile } from '@web/test-runner-commands';
+import { expect } from '@esm-bundle/chai';
+import { waitForElement } from '../../helpers/waitfor.js';
+
+const { default: init } = await import('../../../libs/features/interlinks.js');
+
+describe('Interlinks', () => {
+  it('Interlinks works for english language', async () => {
```

Comment:
nit: since `Interlinks` is used in the describe, you can omit it from the `it` statements. e.g. `it('works for the english language' () => {...})` , `it('works for the korean language', () => {...})`

Reply:
Done